### PR TITLE
fixed binary-search.spec.coffee

### PR DESCRIPTION
**Pull request by ladokp**
@ErikSchierboom: Once again a needed string fix in the test specs.

### CODE REVIEW

File: exercises/practice/binary-search/binary-search.spec.coffee [hunk 1]
```diff
@@ -37,13 +37,13 @@ describe 'Binary Search', ->
       new BinarySearch(array).find 7
     .toThrow new Error 'value not in array'
 
-  xit "a value smaller than the array's smallest value is not found", ->
+  xit 'a value smaller than the array's smallest value is not found', ->
```
Comment (ErikSchierboom) [inline exercises/practice/binary-search/binary-search.spec.coffee:40]:
```suggestion
  xit 'a value smaller than the array\'s smallest value is not found', ->
```

File: exercises/practice/binary-search/binary-search.spec.coffee [hunk 2]
```diff
@@ -37,13 +37,13 @@ describe 'Binary Search', ->
...
     array = [1, 3, 4, 6, 8, 9, 11]
     expect ->
       new BinarySearch(array).find 0
     .toThrow new Error 'value not in array'
 
-  xit "a value larger than the array's largest value is not found", ->
+  xit 'a value larger than the array's largest value is not found', ->
```
Comment (ErikSchierboom) [inline exercises/practice/binary-search/binary-search.spec.coffee:46]:
```suggestion
  xit 'a value larger than the array\'s largest value is not found', ->
```

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]
I would also suggest someone looks at [the test runner](https://github.com/exercism/coffeescript-test-runner) to fix this properly.

File: exercises/practice/binary-search/binary-search.spec.coffee (hunk 1 above)
Comment (ladokp) [reply exercises/practice/binary-search/binary-search.spec.coffee:40]: I should also have read the string's content... -.-

**Comment by ladokp** [comment]
@ErikSchierboom: fixed both strings now.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**